dev/run-tests.py

```diff
@@ -295,13 +295,14 @@ def exec_sbt(sbt_args=()):
 
 def get_hadoop_profiles(hadoop_version):
     """
-    For the given Hadoop version tag, return a list of SBT profile flags for
+    For the given Hadoop version tag, return a list of Maven/SBT profile flags for
```

Comment:
These flags are used in both Maven and SBT, hence this clarification (to see this, grep the file for usages of `get_hadoop_profiles`).